OSDOCS-6754: Network Observability easier configuration (#63276)
openshift/openshift-docs: modules/network-observability-flowcollector-view.adoc

    logTypes: FLOWS                            <3>
    conversationHeartbeatInterval: 30s
  loki:                                       <4>
    lokiStack:

>       name: loki

Author: Ok I was missing `mode: LOKISTACK` as well as `namespace: netobserv`. I added both.

>       name: loki

Author: Oh, I see that PR you linked here with `mode: LOKISTACK`. I guess that is still in progress, so this may or may not go.

>       name: loki

Reviewer: Yes the PR is still WIP

        type: configmap
        name: loki-gateway-ca-bundle
        certFile: service-ca.crt

>         namespace: loki-namespace            <5>

Reviewer: ```suggestion
        # namespace: loki-namespace            <5>
```

Let's comment that by default so the users can still install loki in `netobserv` namespace following the doc and will understand they can optionnaly customize it

Author: Ok yeah that's a good suggestion. I was also thinking, all the examples use the `netobserv` namespace. So I'll make it more clear that different namespace is an option but in our examples we use the same `netobserv` namespace across. And I'll comment that part out of the CRD too, good idea. TY!

<3> The optional specifications `spec.processor.logTypes`, `spec.processor.conversationHeartbeatInterval`, and `spec.processor.conversationEndTimeout` can be set to enable conversation tracking. When enabled, conversation events are queryable in the web console. The values for `spec.processor.logTypes` are as follows: `FLOWS` `CONVERSATIONS`, `ENDED_CONVERSATIONS`, or `ALL`. Storage requirements are highest for `ALL` and lowest for `ENDED_CONVERSATIONS`.
<4> The Loki specification, `spec.loki`, specifies the Loki client. The default values match the Loki install paths mentioned in the Installing the Loki Operator section. If you used another installation method for Loki, specify the appropriate client information for your install.

> <5> If you chose to install Loki in a different namespace, you must specify it in the `spec.loki.tls.caCert.namespace` field. The original certificates copied to the Network Observability instance namespace and are watched for updates. When not provided, the namespace defaults to be the same as "spec.namespace". Similarly, the `spec.exporters.kafka.tls.caCert.namespace` field is available for Kafka installed in a different namespace. 

Reviewer: ```suggestion
<5> If you chose to install Loki in a different namespace, you must specify it in the `spec.loki.tls.caCert.namespace` field. The original certificates are copied to the Network Observability instance namespace and watched for updates. When not provided, the namespace defaults to be the same as "spec.namespace". Similarly, the `spec.exporters.kafka.tls.caCert.namespace` field is available for Kafka installed in a different namespace. 
```

Author: Addressed

> <5> If you chose to install Loki in a different namespace, you must specify it in the `spec.loki.tls.caCert.namespace` field. The original certificates are copied to the Network Observability instance namespace and watched for updates. When not provided, the namespace defaults to be the same as "spec.namespace". Similarly, the `spec.exporters.kafka.tls.caCert.namespace` field is available for Kafka installed in a different namespace. 

Author: I can see this. Agree!

> <5> If you installed Loki in a different namespace, the `spec.loki.tls.caCert.namespace` designates the source namespace where the certificate exists. The original certificates are watched so that the copies are automatically updated when needed. When not provided, the namespace is assumed to be same as "spec.namespace". Similarly, the `spec.exporters.kafka.tls.caCert.namespace` field is available for Kafka installed in a different namespace. 

Reviewer: ```suggestion
<5> If you installed Loki in a different namespace, you must specify it in `spec.loki.tls.caCert.namespace`. The original certificates are copied to Network Observability instance namespace and are watched for updates. When not provided, the namespace is assumed to be same as "spec.namespace". Similarly, the `spec.exporters.kafka.tls.caCert.namespace` field is available for Kafka installed in a different namespace. 
```
